libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -1125,17 +1125,17 @@ int HTTPClient::handleHeaderResponse()
                     if(transferEncoding.equalsIgnoreCase(F("chunked"))) {
                         _transferEncoding = HTTPC_TE_CHUNKED;
                     } else {
-                        return HTTPC_ERROR_ENCODING;
+                        return (_returnCode = HTTPC_ERROR_ENCODING);
```

Comment:
Style request:  Please separate assignment and return here and below, just for sanity's sake.

Reply:
Done!